infra/airgap/scripts/generate-sbom.sh

```
@@ -0,0 +1,296 @@
+    case "${SBOM_FORMAT}" in
+        cyclonedx)
+            sbom_file=$(generate_cyclonedx)
+            generate_vuln_report "${sbom_file}"
+            ;;
+        spdx)
+            sbom_file=$(generate_spdx)
+            generate_vuln_report "${sbom_file}"
+            ;;
+        both)
+            cyclonedx_file=$(generate_cyclonedx)
+            spdx_file=$(generate_spdx)
```

Shellcheck (0.11.0)

[warning] spdx_file appears unused. Verify use (or export if used externally).

(SC2034)